src/main/kotlin/org/hyacinthbots/lilybot/extensions/util/Tags.kt

```diff
@@ -149,6 +150,34 @@ class Tags : Extension() {
 							"${arguments.user?.mention ?: ""}\n**${tagFromDatabase.tagTitle}**\n${tagFromDatabase.tagValue}"
 					}
 				}
+
+				// Log when a message tag is sent to allow identification of tag spammers
+				if (tagFromDatabase.tagAppearance == "message") {
+					getLoggingChannelWithPerms(
+						guild!!.asGuild(),
+						UtilityConfigCollection().getConfig(guild!!.id)!!.utilityLogChannel ?: guild!!.asGuild()
+							.getSystemChannel()!!.id,
+						ConfigType.UTILITY,
+						interactionResponse
+					)?.createMessage {
+						embed {
+							author {
+								name = "Message Tag used"
+								icon = user.asUser().avatar?.url
+							}
+							description = "${user.asUser().mention} (${user.asUser().tag}) used a message tag"
```

Comment:
Let's rebuild this embed to have no description, but a user mention and ID with the user icon/tag in the footer.